sw/host/provisioning/ft/BUILD

```diff
@@ -38,3 +28,12 @@ package(default_visibility = ["//visibility:public"])
     )
     for sku in EARLGREY_SKUS.keys()
 ]
+
+filegroup(
+    name = "ft_all",
+    testonly = True,
```

Comment:
Why testonly?  Because the srcs are testonly, but again, why?

Reply:
It is because all the FPGA exec_envs are marked as "testonly" so it forces this downstream.